pkg/deploy/generator/resources_dev.go

```diff
@@ -318,7 +318,7 @@ enabled=yes
 gpgcheck=yes
 EOF
 
-yum --enablerepo=rhui-rhel-7-server-rhui-optional-rpms -y install azure-cli docker jq libassuan-devel gcc gpgme-devel rh-git29 rh-python36 tmpwatch lttng-usr
+yum --enablerepo=rhui-rhel-7-server-rhui-optional-rpms -y install azure-cli docker jq libassuan-devel gcc rh-git29 rh-python36 tmpwatch lttng-usr
```

Comment:
libassuan-devel and gcc still needed?

Reply:
Need gcc for `go generate`.  If we want to not include it, we can set the environment variable `CGO_ENABLED=0` before building.  
```bash
[cloud-user@bvesel ARO-RP]$ make aro
go generate ./...
# runtime/cgo
exec: "gcc": executable file not found in $PATH
```

Any preference?

Reply:
Looks like in a local test that it's 2.5x slower with `CGO_ENABLED=0`.  If this were only contained to the multistage Dockerfile then I'm easily +1 to this.  I feel it should be reasonable for a local build too but I am not sure how often those builds are done and then what those impacts are.  Since it's a container and just keeping gcc dependency I'd say we just keep it and be done.  I don't have any good reason to remove it from the builder container.